### PR TITLE
Report5

### PR DESCRIPTION
**Pull request by hpnog**

### CODE REVIEW

File: ESOF-docs/report4.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+![alt tag](https://raw.githubusercontent.com/hpnog/jpacman-framework/master/ESOF-docs/test.png)
+
+**Level**
+
+The Level area isn't well covered because there are some problems covering the interactions between colisions in all the map as you can see in the following image :
+
+![alt tag](https://raw.githubusercontent.com/hpnog/jpacman-framework/master/ESOF-docs/level.png)
+
+We talked with the collobaroters of the project about this matter and they told us that this happens because its very difficult to control all the active actors on the map they told us they were still trying to find the ideal solution to solve this prolem and if we can help them they would appreciate it.
```
Comment (JWGmeligMeyling) [inline ESOF-docs/report4.md]: To whom you spoke?

File: ESOF-docs/report4.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+
+**Exception**
+
+Finally we noticed that the project doesnt tests the most part of the exceptions that are trowhn in this program like in this case:
+
+![alt tag](https://raw.githubusercontent.com/hpnog/jpacman-framework/master/ESOF-docs/exception.png) 
+
+
+The most eficient solution to solve this problem is being discussed by all the coloborators by a long time but they still didn´t reached a full agreement to which is the best solution to this problem.
```
Comment (JWGmeligMeyling) [inline ESOF-docs/report4.md]: Which unresolved issue are you referring to exactly?

File: ESOF-docs/report4.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+## Introduction
+
+> Software testability is the degree to which a software artifact (i.e. a software system, software module, requirements- or design document) supports testing in a given test context. If the testability of the software artifact is high, then finding faults in the system (if it has any) by means of testing is easier.
+
+>  _Definition from [Wikipedia]_
+[Wikipedia]: https://en.wikipedia.org/wiki/Software_testability
+
+In this report we will analyse **JPacman Framework**. 
+
+**JPacman** is an open source project developed by DELF College at the United States of America with the goal of teaching students how to do Unitary Tests in a more efficient way.
```
Comment (JWGmeligMeyling) [inline ESOF-docs/report4.md]: We're in NL

File: ESOF-docs/report4.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+During the tests, the compiler performs the heavy lifting when it comes to common bugs, so you can focus on other things and the language is a good match with unitary tests which leads to easily composable code. It is very simple to maintain coverage and to deploying the service. 
+
+For JPacman the usual practice is to have tests for some specific action like create board,move player,capture food,collision,etc.
+
+One of the biggest concerns about the testability of JPacman is the lack of support for handeling Exceptions and some type of Collisions. 
+
+
+As with any project, the project itself plays a big role. That said, for unit testing JPacman´s existing test tools are more than adequate. Integration testing may be more difficult, depending on what you're integrating, but Eclipse supports all the tools to create custom tests that sometimes may be hard to write but that in the end work very well.
```
Comment (JWGmeligMeyling) [inline ESOF-docs/report4.md]: I think you're not familiar with the concept of integration testing. You're not integrating anything with something else, you're testing the integration between the conponents in a system, as opposed to unit testing.

**Comment by hpnog** [comment]
This is not a final version of this report and this Pull Request was a mistake. Sorry...

**Pull request closed without merging**